frontend/src/components/ui/WelcomeModal.jsx

```diff
@@ -1,56 +1,124 @@
-      centered
-    >
-      <Modal.Header>
-        <Modal.Title id="contained-modal-title-vcenter">
-          Mission Briefing
-        </Modal.Title>
-      </Modal.Header>
-      <Modal.Body>
-        <p align="center">
-          Your enemy seeks the quickest path to victory.<br />
-          Alter their trajectory.
-          Bring forth their defeat.
-        </p>
-        {errorMessage && <p className="text-danger">{errorMessage}</p>}
-      </Modal.Body>
-      <Modal.Footer>
-        <Button onClick={loadGame}>Load Game from Clipboard</Button>
-        <Button onClick={hideModal}>New Game</Button>
-      </Modal.Footer>
-    </Modal>
-  );
+
+    return (
+        <Modal
+            show={true}
+            size="lg"
+            aria-labelledby="contained-modal-title-vcenter"
+            centered
+        >
+            <Modal.Header>
+                <Modal.Title id="contained-modal-title-vcenter">
+                    Mission Briefing
+                </Modal.Title>
+            </Modal.Header>
+            <Modal.Body>
+                <p align="center">
+                    Your enemy seeks the quickest path to victory.<br />
+                    Alter their trajectory.
+                    Bring forth their defeat.
+                </p>
+                {loaded && (<table style={{ borderSpacing: '10px', borderCollapse: 'separate' }}>
+                    <thead>
+                        <tr>
+                            <th>Level</th>
+                            <th>Gold</th>
+                            <th>Castle HP</th>
+                            <th>Game Started</th>
+                            <th>Game Saved</th>
+                            <th>Load Game</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        {cloudGames.length ? cloudGames.map((game) => {
+                            const gameData = JSON.parse(atob(game.data));
+                            const { createdAt, updatedAt, level, gold, castleHP } = gameData;
+
+                            return (<tr key={game._id}>
+                                <td>{level}</td>
+                                <td>{gold}</td>
+                                <td>{castleHP}</td>
+                                <td>{new Date(parseInt(createdAt)).toLocaleString()}</td>
+                                <td>{new Date(parseInt(updatedAt)).toLocaleString()}</td>
+                                <td>
+                                    <Button
+                                        onClick={() => {
+                                            setGame(Game.from(game.data));
+                                            hideModal();
+                                        }}
+                                    >
+                                        Load Game
+                                    </Button>
+                                </td>
+                            </tr>)
```

Comment:
Could you make this a table row function? It's a little hard to read inside the ternary block